app/facades/merchant_facade.rb

```diff
@@ -0,0 +1,6 @@
+class MerchantFacade
+  # def self.merchants
+  #   json = MovieService.merchants
+  #   @merchant_info = json[:]
```

Comment:
I know this refactor isn't done yet so I won't insert too many opinions here, but this would be a good spot to use a PORO to hold the Merchant info for easy access in the view.